robocop/reports.py

```diff
@@ -223,8 +226,79 @@ def __init__(self):
         self.name = "version"
         self.description = "Returns Robocop version"
 
-    def add_message(self, *args):
-        pass
-
     def get_report(self) -> str:
         return f"\nReport generated by Robocop version: {__version__}"
+
+
+class TimestampReport(Report):
+    """
+    Report name: ``timestamp``
+
+    Report that returns Robocop execution timestamp.
+    Timestamp follows local time in format of
+    `Year-Month-Day Hours(24-hour clock):Minutes:Seconds ±hh:mm UTC offset` as default.
+
+    Example::
+
+        Reported: 2022-07-10 21:25:00 +0300
+
+    Both of default values, ``timezone`` and ``format`` can be configured by
+    ``-c/--configure`` and ``timestamp:timezone:"<timezone name>"`` and/or ``timestamp:format:"<format string>"``::
+
+        robocop --configure timestamp:timezone:"Europe/Paris" --configure timestamp:format:"%Y-%m-%d %H:%M:%S %Z %z"
+
+    This yields following timestamp report::
+         Reported: 2022-07-10 20:38:10 CEST +0200
```

Comment:
``::`` blocks require one empty line before/after wrapped code
```suggestion
    This yields following timestamp report::

         Reported: 2022-07-10 20:38:10 CEST +0200
```

Reply:
Fixed.